cmd/test-bot.rb

```diff
@@ -65,9 +69,20 @@ def test_bot_args
              description: "Only run the formulae dependents steps."
       switch "--only-cleanup-after",
              description: "Only run the post-cleanup step. Needs `--cleanup`."
+      flag   "--testing-formulae=",
+             description: "Use these testing formulae rather than running the formulae detection steps."
+      flag   "--added-formulae=",
+             description: "Use these added formulae rather than running the formulae detection steps."
+      flag   "--deleted-formulae=",
+             description: "Use these deleted formulae rather than running the formulae detection steps."
+      flag   "--skipped-or-failed-formulae=",
```

Comment:
@MikeMcQuaid (sorry for the late comment) can these use `comma_array` instead of `flag` to avoid the `.split(",")` later on?